distribution/server/src/assemble/LICENSE.bin.txt

```diff
@@ -371,7 +371,8 @@ The Apache Software License, Version 2.0
     - io.netty-netty-transport-native-epoll-4.1.51.Final.jar
     - io.netty-netty-transport-native-unix-common-4.1.51.Final.jar
     - io.netty-netty-transport-native-unix-common-4.1.51.Final-linux-x86_64.jar
-    - io.netty-netty-tcnative-boringssl-static-2.0.30.Final.jar
+    - io.netty-netty-transport-native-kqueue-4.1.51.Final-osx-x86_64.jar
```

Comment:
why do we need kqueue ?

Reply:
Agree, that is not required. Changed accordingly. Thanks.